e2e-tests/playwright/e2e/plugins/application-provider-listener.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { test } from "@playwright/test";
+import { UIhelper } from "../../utils/ui-helper";
+import { Common } from "../../utils/common";
+import { HomePage } from "../../support/pages/home-page";
+
+test.describe("Test ApplicationProvider", () => {
+  let uiHelper: UIhelper;
+
+  test.beforeEach(async ({ page }) => {
+    const common = new Common(page);
+    uiHelper = new UIhelper(page);
+    await common.loginAsGuest();
+  });
+
+  test("Verify that the TestPage is rendered", async ({ page }) => {
+    await page.goto("/application-provider-test-page");
+    await uiHelper.verifyText("application/provider TestPage");
+    await uiHelper.verifyText(
+      "This card will work only if you register the TestProviderOne and TestProviderTwo correctly.",
+    );
+  });
```

Comment:
That the page is rendered is good, but that's not the complete verification.

You should have also a test that clicks on the first + button, and then both card contents should be updated. Only then you know that the provider is setup correctly.

I would expect that this test pass also if you remove the provider.